_data/cn/intro.yml

```diff
@@ -0,0 +1,11 @@
+TC39:
+  title: TC39
+  desc: Ecma International的TC39是一个由JavaScript开发者、实现者、学者等组成的团体，与社区合作维护和发展JavaScript的定义。
```

Comment:
I suggest to add an space (" ") when mixing Chinese character and Latin character. It would bring better visual effects to the reader.

Not good: "发展JavaScript的定义"
Better: "发展 JavaScript 的定义"